pygmt/helpers/decorators.py

```diff
@@ -567,17 +567,6 @@ def new_module(*args, **kwargs):
                     )
                     warnings.warn(msg, category=SyntaxWarning, stacklevel=2)
 
-            # timestamp (U) is deprecated since v0.9.0.
```

Comment:
After this PR, the expected behaviors are: 

- if `timestamp=True` is used, Python will report the parameter is not recognized
- if single-letter parameter `U=True` is used, PyGMT should tell users that `U` is not supported and `Figure.timestamp` should be used.

After removing lines 570-580, parameter `timestamp` is no longer recognized, but single-letter parameter `U` will still be processed. Thus, instead of removing these lines completely, we should change them to something like:

```python
            # timestamp (U)is deprecated since v0.9.0.
            if "U" in kwargs:
                msg = (
                    "Parameter timestamp/U is no longer supported since v0.12.0."
                    "Use Figure.timestamp() instead."
                )
                raise GMTInvalidInput(msg)
```

Reply:
Ah, OK. Thanks! I have tried to change this in the commits https://github.com/GenericMappingTools/pygmt/pull/3045/commits/6977a57302681307239eafca840342a00ff07628, https://github.com/GenericMappingTools/pygmt/pull/3045/commits/1501ccc9291948b6765cb6dd140650ebf1d6e241, and https://github.com/GenericMappingTools/pygmt/pull/3045/commits/42fab3b1e99a8d0ab66d4daca8705190f68c0736. I also changed this in PR #3044 for `xshift` (`X`) and `yshift` (`Y`).